c7n/filters/core.py

```diff
@@ -467,6 +467,10 @@ def get_resource_value(self, k, i):
             # Azure schema: 'tags': {'key': 'value'}
             elif 'tags' in i:
                 r = i.get('tags', {}).get(tk, None)
+                if not r:
+                    # GCP instance objects have tags listed under items key
+                    # which should be checked unless labels are used
```

Comment:
which gcp resources do this?

Reply:
Those are VM instances and instance templates.
https://cloud.google.com/vpc/docs/add-remove-network-tags

Reply:
tags are a legacy construct (single word), labels (k:v) pairs are common use and are analogous to tags in other environments. value filter shouldn't have support for tags. specific usages for tags, can specify that parameter directly in their filters. iotw, this change shouldn't be present.

Reply:
Ok, I have removed the change, thank you for explanations.